src/Microsoft.AspNetCore.Sockets.Client/WebSocketsTransport.cs

```diff
@@ -68,6 +68,7 @@ public WebSocketsTransport(ILoggerFactory loggerFactory)
                     if(receiveResult.MessageType == WebSocketMessageType.Close)
```

Comment:
One thing we talked about that has not been implemented is handling exceptions thrown ReceiveAsync.